docs/user-guide.md

```diff
@@ -227,6 +227,10 @@ Liberty Tools editing assistance provides code completion, diagnostics, and quic
 
 ![Eclipse LSP4MP microprofile-config.properties completion](/docs/user-guide-screenshots/mp_completion.png)
 
+3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix (code action) appears.
```

Comment:
```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick fix link appears. Selecting the quick fix link will provide a list of quick fix options to choose from.
```

Reply:
OR
```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a "Quick Fix" link appears. Click on the "Quick Fix" link to view a list of quick-fix options to choose from.
```

Reply:
Are the words "Quick Fix" visible in the UI? If so I'd recommend bold instead of quotes. If not, would avoid quotes if possible. We try not to use them unless someone is actually being quoted as they interfere with accessibility verification in some contexts. Also, per IBM style, avoid future tense

Reply:
Ok.